src/diffpy/utils/diffraction_objects.py

```diff
@@ -50,7 +50,7 @@ class DiffractionObject:
         The array containing the quantity of q, tth, d values.
     input_xtype : str
         The type of the independent variable in `xarray`. Must be one of {*XQUANTITIES}
-    id : uuid
+    _uuid : uuid
```

Comment:
if we are making this attribute public and giving it a getter, I wonder if we should just call it uuid and remove the underscore?  What do you think?  It may also be less confusing if the attribute name and how it is accessed is the szme?

Reply:
Yes, I agree. It turns out we shall write a separate docstring for `@property` methods as shown below (as done in the latest commit):

```
    @property
    def uuid(self):
        """The unique identifier for the DiffractionObject instance.

        Returns
        -------
        uuid
            The unique identifier of the DiffractionObject instance.
        """
        return self._uuid
```

This way, User does not have to read private properties starting with `_`.

API doc is also clearly marked with `@property`:

<img width="684" alt="Screenshot 2024-12-25 at 11 12 35 PM" src="https://github.com/user-attachments/assets/2b61a7b4-c0bc-4db7-a3d8-7036b2194b34" />